packages/lesswrong/server/languageModels/modGPT.ts

```diff
@@ -261,21 +262,26 @@ async function checkModGPT(comment: DbComment, post: DbPost): Promise<void> {
 }
 
 getCollectionHooks("Comments").updateAsync.add(async ({oldDocument, newDocument}) => {
-  // on the EA Forum, ModGPT checks earnest comments on posts for norm violations
+  // On the EA Forum, ModGPT checks earnest comments on posts for norm violations.
+  // We skip comments by unreviewed authors, because those will be reviewed by a human.
   if (
     !isEAForum ||
     !newDocument.postId ||
     newDocument.deleted ||
+    newDocument.deletedPublic ||
     newDocument.spam ||
+    newDocument.needsReview ||
+    newDocument.authorIsUnreviewed ||
     newDocument.retracted ||
+    newDocument.rejected ||
     newDocument.shortform ||
-    !oldDocument.contents?.originalContents?.data ||
```

Comment:
This wasn't right, we don't want to skip a comment purely because the previous version was blank